public/framework/catalog_cache/cache_loader.tsx

```diff
@@ -253,7 +255,7 @@ export const useLoadToCache = (loadCacheType: LoadCacheType) => {
     startPolling,
     stopPolling: stopLoading,
   } = usePolling<any, any>((params) => {
-    return sqlService.fetchWithJobId(params);
+    return sqlService.fetchWithJobId(params, dataSourceMDSClientId.current);
```

Comment:
does this still trigger request if `dataSourceMDSId` is undefined and `dataSourceMDSClientId.current` is empty string?

Reply:
yup

Reply:
do we want to trigger it when it's undefined / empty string?